hack/update-codegen.sh

```diff
@@ -24,8 +24,9 @@ CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${REPO_ROOT_DIR}; ls -d -1 ./vendor/k8s.io/code-
 
 # Generate based on annotations
 go generate ./pkg/... ./cmd/... ./contrib/gcppubsub/pkg/...
+go generate ./pkg/... ./cmd/... ./contrib/camel/pkg/...
```

Comment:
It looks like this removed `./contrib/gcppubsub/pkg/...`, rather than adding to the line?

Reply:
No, I don't see it removed. Both are executed btw..

Reply:
Oh, I missed that this wasn't a `-` of the previous line. You should be able to add `./contrib/camel/pkg/...` to the previous line, rather than generating `./pkg/...` and `./cmd/...` twice.

Reply:
Uh, my bad